iped-app/src/main/java/iped/app/timelinegraph/IpedChartsPanel.java

```diff
@@ -752,7 +752,7 @@ public void tableChanged(TableModelEvent e) {
         }
         if (!dataSetUpdated.getAndSet(true)) {
             new Thread(populateEventNames).start();
-            // ipedTimelineDatasetManager.startBackgroundCacheCreation();
+            ipedTimelineDatasetManager.startCacheCreation();
```

Comment:
Hi @patrickdalla, I noticed this line was enabled in this PR. Is it intentional and needed? Maybe it could cause some issue with the changes of last merged PR...

Reply:
This seems to have being pushed by mistake, sorry. I think I put it for test purpose as the last PR wasn't applyed yet.

Reply:
I have noticed it now, and also noted that Felipe have done that in first commit. I think that, as the last PR was't applyed yet, he needed this for some testing reason alone and pushed by mistake.
You have changed the name from startBackgroundCacheCreation to startCacheCreation, right?

Reply:
> You have changed the name from startBackgroundCacheCreation to startCacheCreation, right?

Yes, since it is synchronous, I thought it would be a better name.